.github/workflows/build_docker.yml

```diff
@@ -10,8 +10,17 @@ jobs:
   build-with-docker:
     runs-on: ubuntu-20.04
     steps:
+      - name: Remove unnecessary files
+        run: |
+          sudo rm -rf /usr/share/dotnet
```

Comment:
Got this from https://github.com/actions/runner-images/issues/2840#issuecomment-1284059930